README.md

```diff
@@ -6,107 +6,141 @@
-### Sequential and Session-based recommendation
-Traditional recommendation algorithms usually ignore the temporal dynamics and the sequence of interactions when trying to model user behaviour. Generally, the next user interaction is related to the sequence of the user's previous choices. In some cases, it might be even a repeated purchase or song play. User interests might also suffer from the interest drift, as preferences might change over time. Those challenges are addressed by the **sequential recommendation** task.
-A special case of sequential-recommendation is the **session-based recommendation** task, where you have only access to the short sequence of interactions within the current session. This is very common in online services like e-commerce, news and media portals where the user might choose to browse anonymously (and due to GDPR compliance no cookies are collected), or because it is a new user. This task is also relevant for scenarios where users' interests change a lot over time depending on the user context or intent, so leveraging the current session interactions is more promising than old interactions to provide relevant recommendations.
+## Installation
 
-To deal with sequential and session-based recommendation, many sequence learning algorithms previously applied in machine learning and NLP research have been explored for RecSys, based on k-Nearest Neighbors, Frequent Pattern Mining, Hidden Markov Models, Recurrent Neural Networks, and more recently neural architectures using the Self-Attention Mechanism and the Transformer architectures.
+You can install Transformers4Rec with Pip, Conda, or run a Docker container.
 
-Differently from Transformers4Rec, existing frameworks for such tasks are generally focused for research, accept only sequence of item ids as input and do not provide a modularized and scalable implementation for production usage.
+### Installing Transformers4Rec Using Pip
 
+You can install Transformers4Rec with the functionality to use the GPU-accelerated Merlin dataloader.
+Installation with the dataloader is highly recommended for better performance.
+Those components can be installed as optional arguments for the `pip install` command.
 
-## Installation
+To install Transformers4Rec using Pip, run the following command:
 
-### Installing with pip
+```shell
+pip install transformers4rec[pytorch,nvtabular]
+```
 
-You can install Transformers4Rec with the functionality to use the GPU-accelerated NVTabular dataloader.
-Installation with the dataloader is highly recommended for better performance.
-Those components can be installed as optional args for the pip install package. Note that installation NVTabular with `pip` supports only CPU version of NVTabular for now.
+> Be aware that installing Transformers4Rec with `pip` only supports the CPU version of NVTabular because `pip` does not install cuDF.
+> The GPU capabilities of NVTabular are available by using the Docker container or by installing
```

Comment:
we should replace `NVTabular` with `Merlin dataloader` following the changes of #547

Reply:
Excellent.  Thank you for helping me identify the change from NVT to the Merlin Dataloader library.